Tools/Frame_params/ModalAI/AutonomyDevKit.parm

```diff
@@ -6,6 +6,12 @@ FLTMODE1         5
 FLTMODE4         2
 FLTMODE_CH       6
 
+# motor ordering
+SERVO1_FUNCTION 34
+SERVO2_FUNCTION 35
+SERVO3_FUNCTION 33
+SERVO4_FUNCTION 36
```

Comment:
I guess this is not equivalent to one of our existing frame types?

Reply:
its not